interfaces/console/src/app/console/home/page.tsx

```diff
@@ -46,74 +51,56 @@ const StatusCard = dynamic(() => import('@/components/StatusCard'), {
 const networkNodesLoading = false;
 export default function Page() {
-  const { network } = useAppContext();
-  // const [filterState, setFilterState] = useState<NodeStatusEnum>(
-  //   NodeStatusEnum.Undefined,
-  // );
-  // const { data: networkRes, loading: networkLoading } = useGetSitesQuery({
-  //   fetchPolicy: 'no-cache',
-  //   variables: {
-  //     networkId: network.id,
-  //   },
-  //   onError: (error) => {
-  //     setSnackbarMessage({
-  //       id: 'home-sites-err-msg',
-  //       message: error.message,
-  //       type: 'error' as AlertColor,
-  //       show: true,
-  //     });
-  //   },
-  // });
+  const { network, setSnackbarMessage } = useAppContext();
+  const { data: sitesRes, loading: sitesLoading } = useGetSitesQuery({
+    fetchPolicy: 'no-cache',
+    variables: {
+      networkId: network.id,
+    },
+    onError: (error) => {
+      setSnackbarMessage({
+        id: 'home-sites-err-msg',
+        message: error.message,
+        type: 'error' as AlertColor,
+        show: true,
+      });
+    },
+  });
+
+  const { data: nodesData, loading: nodesLoading } = useGetNodesLocationQuery({
+    fetchPolicy: 'cache-and-network',
+    onError: (error) => {
+      setSnackbarMessage({
+        id: 'home-nodes-err-msg',
+        message: error.message,
+        type: 'error' as AlertColor,
+        show: true,
+      });
+    },
+  });
 
   // const { data: statsRes, loading: statsLoading } = useGetStatsMetricQuery({
   //   client: getMetricClient("", ""),
   //   fetchPolicy: 'cache-and-network',
   // });
 
-  // const { data: nodesLocationData, loading: nodesLocationLoading } =
-  //   useGetNodesLocationQuery({
-  //     fetchPolicy: 'cache-first',
-  //     variables: {
-  //       data: {
-  //         nodeFilterState: filterState,
-  //         networkId: network.id,
-  //       },
-  //     },
-  //   });
-
-  // const { data: networkNodes, loading: networkNodesLoading } =
-  //   useGetNodesByNetworkQuery({
-  //     fetchPolicy: 'cache-and-network',
-  //     variables: {
-  //       networkId: network.id,
-  //     },
-  //     onError: (error) => {
-  //       setSnackbarMessage({
-  //         id: 'home-network-nodes-err-msg',
-  //         message: error.message,
-  //         type: 'error' as AlertColor,
-  //         show: true,
-  //       });
-  //     },
-  //   });
-
   return (
     <Grid container rowSpacing={2} columnSpacing={2}>
-      <Grid xs={12}>
+      <Grid size={12}>
         <NetworkStatus
           title={
             network.name
               ? `${network.name} is created.`
               : `No network selected.`
           }
-          subtitle={network.name ? 'No node attached to this network.' : ''}
+          subtitle={network.name ? '' : ''}
           loading={false}
           availableNodes={undefined}
           statusType="ONLINE"
           tooltipInfo="Network is online"
         />
       </Grid>
-      <Grid xs={12}>
+      <Grid size={12}>
```

Comment:
Why do you use `size` instead of `xs`? I believe `size` is deprecated.

Reply:
No its other away around. Grid is deprecated we need to gradually update Grid to Grid v2
https://mui.com/material-ui/migration/migration-grid-v2/